swag_client/backend.py

```diff
@@ -26,10 +26,8 @@ def validate(item, namespace='accounts', version=2, context=None):
     """
     if namespace == 'accounts':
         if version == 2:
-            schema = v2.AccountSchema(strict=True, context=context)
-            return schema.load(item).data
-        elif version == 1:
```

Comment:
Is V1 being removed? I see some updates to the V1 schema.

Reply:
i removed logic that deprecates v1 schema. This is only marshmallow upgrades for now